pkg/search/scoped/context.go

```diff
@@ -13,6 +13,7 @@ import (
 // Scope hold an id and scope level for scoping searches.
 type Scope struct {
 	ID     string
+	IDs    []string
```

Comment:
I wonder if we need both `ID` and `IDs` can we have just `IDs`

Reply:
Oh we could absolutely just use IDs.  I started down that path, but the blast radius was just too large.  In the interest of the progress I need to make on the feature, I chose the quicker path.  

So I wrote this:  https://issues.redhat.com/browse/ROX-28813